### PR TITLE
docs(testing): add x-priority to jest package

### PR DESCRIPTION
**Pull request by barbados-clemens**
## Related Issue(s)

Fixes #

### CODE REVIEW

**Comment by barbados-clemens** [comment]
Forgot to do the generators. adding now

File: docs/generated/packages/jest/executors/jest.json [hunk 1]
```diff
@@ -105,7 +108,8 @@
       },
       "passWithNoTests": {
         "description": "Will not fail if no tests are found (for example while using `--testPathPattern`.) (https://jestjs.io/docs/cli#--passwithnotests)",
-        "type": "boolean"
+        "type": "boolean",
+        "x-priority": "important"
```
Comment (MaxKless) [inline docs/generated/packages/jest/executors/jest.json:112]: Is this really something folks will want to set almost all of the time they use this executor? 
To me it seems like `passWithNoTests` gets enabled as soon as there happens to be no tests which isn't that often, right? Most generators scaffold sample tests

File: docs/generated/packages/jest/executors/jest.json [hunk 2]
```diff
@@ -166,15 +170,17 @@
       "updateSnapshot": {
```
Comment (MaxKless) [inline docs/generated/packages/jest/executors/jest.json:170]: Same here. Obviously you're the expert so I'll trust your judgement. But I would think that folks would rather use `-u` when they want to update snapshots and not set it every time...

File: docs/generated/packages/jest/executors/jest.json [hunk 3]
```diff
@@ -166,15 +170,17 @@
...
+        "x-priority": "important"
       },
       "useStderr": {
         "description": "Divert all output to stderr.",
         "type": "boolean"
       },
       "watch": {
         "description": "Watch files for changes and rerun tests related to changed files. If you want to re-run all tests when a file has changed, use the `--watchAll` option. (https://jestjs.io/docs/cli#--watch)",
-        "type": "boolean"
+        "type": "boolean",
```
Comment (MaxKless) [inline docs/generated/packages/jest/executors/jest.json:182]: See the guidelines that say `watch` shouldn't be `important`

**Review by MaxKless — CHANGES_REQUESTED** [review]

File: docs/generated/packages/jest/executors/jest.json [hunk 4]
```diff
@@ -70,7 +70,8 @@
       },
       "ci": {
         "description": "Whether to run Jest in continuous integration (CI) mode. This option is on by default in most popular CI environments. It will prevent snapshots from being written unless explicitly requested. (https://jestjs.io/docs/cli#--ci)",
-        "type": "boolean"
+        "type": "boolean",
+        "x-priority": "important"
```
Comment (MaxKless) [inline docs/generated/packages/jest/executors/jest.json:74]: unless there's some specific reason here, I think `ci` shouldn't be `important` either. it is set by default and kind of a generic option.

**Review by MaxKless — APPROVED** [review]

**Pull request merged**